pom.xml

```diff
@@ -194,6 +199,16 @@
                             </execution>
                         </executions>
                     </plugin>
+                </plugins>
+            </build>
+        </profile>
+        <profile>
+            <id>coveralls</id>
```

Comment:
Coveralls failed if cobertura didn't generate xml report. Splitting provides a possibilty for trigger (coveralls works now only if report is existed)